doc/nvim-tree-lua.txt

```diff
@@ -734,11 +734,16 @@ Window / buffer setup.
 
         *nvim-tree.view.width.min*
           Minimum dynamic width.
-          Type: `string | number | function`, Default: `30`
+            Type: `string | number | function`, Default: `30`
 
         *nvim-tree.view.width.max*
           Maximum dynamic width, -1 for unbounded.
-          Type: `string | number | function`, Default: `-1`
+            Type: `string | number | function`, Default: `-1`
+
+        *nvim-tree.view.width.padding*
+          dynamic padding when resizing. Useful for custom status-column
+          styles.
+            Type: `number`, Default: `3`
```

Comment:
should be the right docs, padding only really supports a number

Reply:
It can be a function, even if just for the sake of consistency.

You might wish to increase padding as you increase max, to "balance things out".